src/app/containers/StoryPromo/index.jsx

```diff
@@ -1,18 +1,87 @@
 import React, { Fragment, useContext } from 'react';
+import moment from 'moment';
 import { shape } from 'prop-types';
+import MediaIndicator from '@bbc/psammead-media-indicator';
 import StoryPromoComponent, {
   Headline,
   Summary,
   Link,
 } from '@bbc/psammead-story-promo';
 import Timestamp from '@bbc/psammead-timestamp-container';
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
 import { storyItem } from '../../models/propTypes/storyItem';
 import StoryPromoFigure from './Figure';
 import { ServiceContext } from '../../contexts/ServiceContext';
 import deepGet from '../../lib/json/deepGet';
+import formatDuration from '../../lib/utilities/formatDuration';
+
+const buildMediaIndicator = (item, mediaTranslations) => {
```

Comment:
Can these go in a new file or 2? its getting pretty big, probably would be nice to unit test them individually too maybe?